src/apps/marketplace/src/app/view/CustomApp.js

```diff
@@ -54,6 +59,19 @@ function LoadApp(props) {
     }
   }, [frame.current, app]);
 
+  const handleZestyDAMRequest = (event) => {
```

Comment:
the same state, handler, useeffect and router wrapping the media component exists within the `CustomApp` and `FreestyleWrapper` components. Would it make sense to extract these out into a High order Component. Which could then wrap any other component.